e2e-tests/cypress/integration/common/appeal-submission-to-horizon.js

```diff
@@ -351,6 +352,39 @@ Then('the associated documents will be available for the case worker to review',
   }
 });
 
+Then('the LPA will receive an email notification of the appeal', () => {
+  if (queueValidationEnabled && notifyValidationEnabled) {
+    cy.task('getLastFromQueue').then((actualMessage) => {
+      const templateId = Cypress.env(
+        'SRV_NOTIFY_APPEAL_SUBMISSION_RECEIVED_NOTIFICATION_EMAIL_TO_LPA_TEMPLATE_ID',
+      );
+      const validationUrl = `${notifyValidationBaseUrl}notifications?template_id=${templateId}&reference=${actualMessage.appeal.id}`;
+
+      cy.request(validationUrl).then((response) => {
+        expect(response.body).to.have.length(1);
+
+        // this changes every run and is not part of the real implementation anyway.
+        delete response.body[0].id;
+
```

Comment:
not critical, but there are some examples lying around the place of doing this with a jest matcher.. rather than deleting things out of the response you can expect `any(String)`

Reply:
<img width="2048" alt="Screenshot 2021-04-20 at 17 00 55" src="https://user-images.githubusercontent.com/6329541/115428167-086a1d00-a1fa-11eb-90dc-adda83bbf69e.png">

Yeah - so I tried this earlier, amidst a million other things going wrong with mocking stuff, so didn't spend a great deal of time on it. 

The way I saw it was the issue was I'm checking deep equality, but then expecting one thing not to be exactly equal.

Any pointers appreciated if I'm missing something hugely obvious - if not, will take a look again tomorrow.

Reply:
Issue addressed as suggested:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/955/files#diff-df0f8c8e054f59eb16788a3026f3bd733f860f66540a21b3b472d09a475c3ed3R378